utils.go

```diff
@@ -62,6 +62,58 @@ func DecodeLabel(image imgutil.Image, label string, v interface{}) error {
 	return nil
 }
 
+// RemoveStagePrefixes creates a slice with the build: or run: style prefixes removed.
+func RemoveStagePrefixes(mixins []string) []string {
+	var result []string
+	for _, m := range mixins {
+		s := strings.SplitN(m, ":", 2)
+		if len(s) == 1 {
+			result = append(result, s[0])
+		} else {
+			result = append(result, s[1])
+		}
+	}
+	return result
+}
+
+// FromSlice converts the given slice to a set in the form of unique keys in a map.
+// The value associated with each key should not be relied upon. A value is present
+// in the set if its key is present in the map, regardless of the key's value.
+// from: https://github.com/buildpacks/pack/blob/main/internal/stringset/stringset.go
```

Comment:
The 2 methods with this comment were pulled from pack. I wasn't sure if there is a better place for these to exist. I didn't think we'd want to reference pack as a library :)

Reply:
I think we can leave them here but we shouldn't export them.

Honestly I am not sure we need to 
a) describe the `FromSlice` behavior in quite as much detail in the comment here; It is common to use a `map[T]struct{}` to represent a set in golang.
b) have a `FromSlice` func at all? I am happy to leave it if we don't export it but I don't necessarily see this standard `for` loop as the type of duplication that needs to be DRYed up.